scripts/install-cni.sh

```diff
@@ -25,10 +25,25 @@ if [ -z "${pod_cidr:-}" ]; then
 fi
 
 if [ -w /host/etc/cni/net.d ]; then
-  cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#podCidr#${pod_cidr:-}#g")
+  cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#@podCidr#${pod_cidr:-}#g")
```

Comment:
Also, as of now, we should not generate CNI spec if ENABLE_CALICO_NETWORK_POLICY is true.